### PR TITLE
Feature/sylius 1.9

### PR DESCRIPTION
**Pull request by BartoszWojdalowicz**
- upgrade to sylius 1.9 
- change sylius_grid from https://github.com/BitBagCommerce/SyliusProductBundlePlugin/issues/9
- add to readme -  how to make yaml config instead xml
- README.md new layout  
- add github actions
- code cleaning

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,42 +6,46 @@
...
+        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
+        "vimeo/psalm": "4.6.4"
     },
     "conflict": {
+        "api-platform/core": "^2.6",
         "symfony/symfony": "4.1.8",
         "symfony/browser-kit": "4.1.8",
         "symfony/dependency-injection": "4.1.8",
         "symfony/dom-crawler": "4.1.8",
         "symfony/routing": "4.1.8"
```
Comment (piotrkardasz) [inline composer.json:47]: These conflict dependencies probably should be removed.

File: composer.json [hunk 2]
```diff
@@ -6,42 +6,46 @@
...
+        "phpunit/phpunit": "^9.5",
         "sensiolabs/security-checker": "^6.0",
-        "sylius-labs/coding-standard": "^3.0",
-        "symfony/browser-kit": "^4.4",
-        "symfony/debug-bundle": "^4.4",
-        "symfony/dotenv": "^4.4",
-        "symfony/intl": "^4.4",
-        "symfony/web-profiler-bundle": "^4.4",
-        "symfony/web-server-bundle": "^4.4"
+        "sylius-labs/coding-standard": "^3.1",
```
Comment (piotrkardasz) [inline composer.json:33]: `vendor/bin/ecs check src` doesn't work with the current dependency.  Do you mean  `"sylius-labs/coding-standard": "^4.0"`?

File: src/Controller/OrderItemController.php [hunk 3]
```diff
@@ -98,16 +98,12 @@ public function addProductBundleAction(Request $request): Response
...
-            ->setTemplate($configuration->getTemplate(CartActions::ADD . '.html'))
-        ;
-
-        return $this->viewHandler->handle($configuration, $view);
+        return $this->render(
+            $configuration->getTemplate(CartActions::ADD . '.html'),
+            ['configuration' => $configuration,
+            $this->metadata->getName() => $orderItem,
+            'form' => $form->createView(),]
+        );
```
Comment (piotrkardasz) [inline src/Controller/OrderItemController.php:106]: Fix lint like e.g.
```php
        return $this->render(
            $configuration->getTemplate(CartActions::ADD . '.html'),
            [
                'configuration' => $configuration,
                $this->metadata->getName() => $orderItem,
                'form' => $form->createView(),
            ]
        );
```
You can also use shortcut `Ctrl + Alt + L` when using PhpStorm

File: composer.json (hunk 2 above)
Comment (BartoszWojdalowicz) [reply composer.json:33]: 3.1 version is from plugin-skeleton. when I try on 4.0 version,  gh actions don't build correctly

File: tests/Application/config/packages/doctrine_migrations.yaml [hunk 4]
```diff
@@ -1,5 +1,4 @@
 doctrine_migrations:
-    dir_name: "%kernel.project_dir%/src/Migrations"
+    migrations_paths:
+        'App\Migrations': "%kernel.project_dir%/src/Migrations"
```
Comment (piotrkardasz) [inline tests/Application/config/packages/doctrine_migrations.yaml:3]: Original skeleton file looks like this:
```yaml
doctrine_migrations:
    storage:
        table_storage:
            table_name: sylius_migrations

```

**Comment by piotrkardasz** [comment]
I create a fork of your branch and not all configuration files are the same as in skeleton. It is possible to make compatibility for both versions of Sylius 1.8 and 1.9 🔥

**Comment by BartoszWojdalowicz** [comment]
sylius 1.8 added, https://github.com/BartoszWojdalowicz/SyliusProductBundlePlugin/actions/runs/777052496

File: src/Controller/OrderItemController.php [hunk 5]
```diff
@@ -45,7 +45,8 @@ public function __construct(
         Controller\ResourceUpdateHandlerInterface $resourceUpdateHandler,
         Controller\ResourceDeleteHandlerInterface $resourceDeleteHandler,
         MessageBusInterface $messageBus
-    ) {
```
Comment (leszczuu) [inline src/Controller/OrderItemController.php:48]: Is this ECS change? I think PSR says 
'When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.'
 
https://www.php-fig.org/psr/psr-12/

Comment (BartoszWojdalowicz) [reply src/Controller/OrderItemController.php:48]: it was probably lint extension in phpstorm. i will change it back.

**Review by leszczuu — APPROVED** [review]

**Review by piotrkardasz — APPROVED** [review]

**Pull request merged**